src/components/clusterprovider/common/cacheinfo.ts

```diff
@@ -0,0 +1,53 @@
+import { getMinikubeShowInfoState } from "../../config/config";
+
+export enum ShowInformationOptions {
+    Install = "Install",
+    InstallFortNightly = "Remind Fortnightly",
+    InstallHalfYearly = "Remind Half Yearly",
```

Comment:
Would it make sense for these to read:
- "Remind me in 2 weeks"
- "Remind me in 6 months"
?